internal/api/external.go

```diff
@@ -76,6 +97,7 @@ func (a *API) ExternalProviderRedirect(w http.ResponseWriter, r *http.Request) e
 		Provider:    providerType,
 		InviteToken: inviteToken,
 		Referrer:    redirectURL,
+		OAuthID:     oauthID,
```

Comment:
Given this is used internally, I would probably choose a different name more indicative of the object it represents... Something like `FlowState`?

Reply:
Okay, will rename OAuthState to FlowState throughout